chart/linkerd-buoyant/README.md

```diff
@@ -0,0 +1,37 @@
+# linkerd-buoyant
+
+![Version: 0.2.0](https://img.shields.io/badge/Version-0.2.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.2.0](https://img.shields.io/badge/AppVersion-0.2.0-informational?style=flat-square)
+
+The Linkerd Buoyant extension connects your Linkerd-enabled Kubernetes cluster to Buoyant Cloud, the global platform health dashboard for Linkerd.
+
+**Homepage:** <https://buoyant.io/cloud>
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| Buoyant developers | info@buoyant.io | https://buoyant.io/cloud |
+
+## Source Code
+
+* <https://github.com/BuoyantIO/linkerd-buoyant>
+
+## Requirements
+
+Kubernetes: `>=1.15.0-0`
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| accountDownloadKey | string | `nil` | The download key for your Buoyant Cloud account. Required. |
+| accountID | string | `nil` | The ID for your Buoyant Cloud account. Required. |
+| accountKey | string | `nil` | The key for your Buoyant Cloud account. Required. |
+| agent.logLevel | string | `"debug"` | Log level for the buoyant-cloud-agent pod |
+| clusterName | string | `nil` | The name of your cluster, as it should appear in Buoyant Cloud. Required |
+| installNamespace | bool | `true` | Set to false when installing Linkerd in a custom namespace. If so, make sure you add the required annotations and labels, as depicted in the namespace.yaml template |
```

Comment:
Infact I kind of take that back. For local developement, we install a docker-host manifest so we can connect to the docker-compose env from the kind cluster, so we already have a namespace. So yes, this flag is needed actually

Reply:
Oops, I'll fix that "linkerd" reference to "linkerd-buoyant agent". More context about this flag: turns out since Helm v3 the best practice is to actually have the templates be namespace-agnostic, and set it with the `--namespace` flag in `helm install`. Mainly because the Helm secret holding the config will live under that same namespace. If we hard-code the namespace then `helm install` will unfortunately put that secret in the default namespace, which is not desirable for obvious reasons.

BUT given we require specific metadata on this namespace, it's just easier for folks if we create it, leaving the option for them to manage it themselves if they're concerned about the secret location.